LibTrixi.jl/Project.toml

```diff
@@ -25,3 +25,9 @@ PrecompileLowStorage = true
 PrecompileNoSpecialize = false
 PrecompileNonStiff = true
 PrecompileStiff = false
+
+[extras]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[targets]
+test = ["Test"]
```

Comment:
```suggestion
```
Instead, create a `LibTrixi.jl/test/Project.toml` file with all relevant entries.

Reply:
I read from the manual that either way is fine. And as I did not fully understand the second approach, I chose the first.

I now copied `Project.toml` to `test/` and added `test` there. Is this how it is supposed to be?